bundler.d/graphql.rb

```diff
@@ -0,0 +1,4 @@
+group :graphql do
```

Comment:
This group implies it's optional but my reading of the initializer is that it's not. Is my understanding correct?

Reply:
Yeah, basically it does not make sense to run the app without this gem.

Reply:
In that case `Gemfile` is the logical place for this.

Reply:
Ok, I'll change this back.